test/functionalTest/cases/0000_ipv6_support/ipv4_ipv6_both.test

```diff
@@ -31,10 +31,10 @@ accumulatorStart --pretty-print :: ${LISTENER2_PORT}
 
 --SHELL--
 echo "1: ++++++++++++++++++++"
-orionCurl --url "/version" --host "[::1]" -g 2> /dev/null
+orionCurl --url "/version" --noPayloadCheck --host "[::1]" -g 2> /dev/null
 
 echo "2: ++++++++++++++++++++"
-orionCurl --url "/version" --host "127.0.0.1" -g 2> /dev/null
+orionCurl --url "/version" --noPayloadCheck --host "127.0.0.1" -g 2> /dev/null
```

Comment:
I have a feeling of veja-vu ...
Why do we need --noPayloadCheck here ?

Reply:
Becasuse since this PR the order of the fields in GET /version response has become relevant.

Reply:
OK,

NTC